pkg/kubelet/kubelet.go

```diff
@@ -424,6 +424,12 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		LowThresholdPercent:  int(kubeCfg.ImageGCLowThresholdPercent),
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.ImageMaximumGCAge) {
```

Comment:
Could you look into adding a unit test for this?  Kubelet has a corresponding unit test in this file so if possible it would be nice to verify the behavior of the feature toggle.

Reply:
what are you hoping to be tested for @kannon92 ? the MaxAge variable in the GC manager is set?

Reply:
Yea, I normally would like to make sure that if someone turns off the feature does it revert to old behavior.  And if its turned on, does it work as intended?

It's not a big deal though.

Reply:
SGTM.